plane/source/docs/takeoff-landingpage.rst

```diff
@@ -0,0 +1,55 @@
+.. _takeoff-landingpage:
+
+=======
+Takeoff
+=======
+
+Taking the plane off can be accomplished in many ways: Rolling Takeoff, Hand Launch, Bungee, or VTOL (Verical Takeoff and Landing) if a :ref:`quadplane-support`. And it can be done manually or automatically.
+
+Rolling Takeoff
+===============
+
+In this case, the plane has landing gear with steerable nose gear, usually coupled to the rudder control and the plane is usually launched from a smooth grass or paved surface. Once aligned nose into the wind, throttle is gradually increased to takeoff value (usually full throttle) while the aircraft direction is controlled by rudder. Once sufficient airspeed to lift off is obtained, slight up elevator will rotate the nose to climb attitude.
+
+MANUAL or FBWA modes are most commonly used during takeoffs.
+
+For automatic takeoffs see :ref:`automatic-takeoff` section.
+
+Hand Launch
+===========
+
+Many planes do not have landing gear, or a smooth runway is not available. Hand launching is a very common takeoff method, but also can be very frustrating if not setup and executed properly.
+
+The keys to a successful hand launch are :
+
+- :ref:`Proper CG <guide-center-of-gravity>` 
+- Having flying surfaces centered and neutral at neutral RC stick positions
+- Having the "level" calibration setup such that the level horizon (0 deg pitch) in FBWA mode results in the wing chord line being a few degrees positive, which will result in a positive angle of attack in "level" flight, the normally cruising attitude for a plane.
+- Using FBWA mode or :ref:`automatic-takeoff` . While launching in MANUAL can be done, using a stabilized mode will eliminate a lot of pilot effort to overcome trim issues (see Tip at end of this page). If using FBWA mode, a few degrees of up elevator (~10%) during the launch will result in an automatic climb. Add this via a switch or stick pressure. Do not change TX trim positions used during RC calibration since it affects all other modes of flight.
+- Throttle up and firmly throw the plane level, not nose pointed up. Common mistakes are pointing the nose up, throwing it toward the ground, or rotating your wrist like a football throw. In the first second or so, the plane has low airspeed and the flying surfaces have no authority to correct non-level attitude until it accelerates to flying speed.
```

Comment:
I'd suggest throwing a few degrees nose-up, otherwise it's common to hit the ground.

Reply:
again, a newbie cant judge a few degrees...we launch as you suggest, but trying for dead level will seldom result in a rebuild if CG is correct..and btw, something like a very heavy MiniTalon wont tolerate even a few degrees..again, the conservative approach here

Reply:
I just don't have the same experience at all; I've launched many "overweight" aircraft that barely made it off the ground, and if they hadn't maintained the 30-degrees of pitch; they would be in pieces on the turf. The "level" acceleration that is gained by a lower takeoff AOA is almost never worth it IMO.